src/rest/api-response.js

```diff
@@ -16,6 +18,13 @@ class ApiResponse {
     }
     return _.isEqual(this.body, body);
   }
+
+  hasMatchingSchema(schema) {
```

Comment:
I think we should either throw an error here or return nothing. Then do try - catch in step definition, so we can either return rejected promise or resolved one. This will make errors much more readable.